chcemvediet/apps/accounts/admin.py

```diff
@@ -44,6 +47,10 @@ class ProfileAdmin(admin.ModelAdmin):
                 short_description=u'Undecided E-mails',
                 admin_order_field=u'undecided_emails_count',
                 ),
+            decorate(
+                lambda o: admin_obj_format(o, u'Log in', link=u'login_as'),
```

Comment:
@martinmacko47 V tomto pripade by bolo fajn keby bol `href target=_blank`, ale to bude potrebne prerobit `admin_obj_format`.